app/views/tag/_tags.html.erb

```diff
@@ -26,7 +26,7 @@
           data-trigger="focus"
           data-count=0 
           data-placement="top" 
-          data-content="<p style='text-align:center;'><a href='/tag/<%= tag.name %>'><%= Tag.tagged_node_count(tag.name) || 0 %> notes</a> - <a href='/contributors/<%= tag.name %>'><%= Tag.contributors(tag.name).count %> people <br></a></p><% if tag.location_tag? %><p><%= translation('tag.location.tooltip') %></p><% end %> <p style='text-align:center;font-size:12px;'><%if tag.description %><%= tag.description %> |<% end %> created by <a href='/profile/<%= tag.try(:author).try(:username) %>'><%= tag.try(:author).try(:username) %></a> <%= time_ago_in_words(Time.at(tag.date)) %> ago </p><div class='text-center'><a href='/subscribe/tag/<%= tag.name %>' class='btn btn-primary'>Follow</a></div>" 
+          data-content="<p style='text-align:center;'><a href='/tag/<%= tag.name %>'><%= Tag.tagged_node_count(tag.name) || 0 %> notes</a> - <a href='/contributors/<%= tag.name %>'><%= Tag.contributors(tag.name).count %> people <br></a></p><% if tag.location_tag? %><p><%= translation('tag.location.tooltip') %></p><% end %> <p style='text-align:center;font-size:12px;'><%if tag.description %><%= tag.description %> |<% end %> created by <a href='/profile/<%= tag.try(:author).try(:username) %>'><%= tag.try(:author).try(:username) %></a> <%= time_ago_in_words(Time.at(tag.date)) %> ago </p><% if !(tag.name.include? ':') %><div class='text-center'><a href='/subscribe/tag/<%= tag.name %>' class='btn btn-primary'>Follow</a></div><% end %>" 
```

Comment:
Gosh these long lines are hard to read, huh? I asked for some input but wasn't able to get much back, as some people are on vacation right now. I say this is a small enough change that let's do it, and we'll be able to trace it back and revert the change later if it's a problem. Thank you!

Reply:
One thing is, i see that the `...` menu only offers an "unfollow" option. Let's get that to switch to "Follow" if you haven't yet -- that way there will always be a way to do it even if it's not strongly advertised via the visible button. You can then always look in the menu. How does that sound, for adding to this PR, so that we don't get in a position where it's much harder to follow if you really need to?

![image](https://user-images.githubusercontent.com/24359/124617957-5c71be80-de45-11eb-943f-2b3a0dfe78ca.png)